### ISSUE TITLE
Overriding the default TheHeader.view component does not work

### ISSUE THREAD

**Issue by adilshehzad786** [nuxt/content#774]
### Version
@nuxt/content-theme-docs": "^0.8.0",
"nuxt": "^2.14.7",

### Reproduction Link

[GitPod ](https://gitpod.io/#https://gitlab.com/grey-software/onboarding/-/tree/investigate-component)
### Steps to reproduce

- yarn 
-  yarn dev
### What is Expected?

**TheHeader.vue** Component Should be working.

```
<template>
  <header
    class="sticky top-0 left-0 right-0 z-30 bg-light-elevatedSurface dark:bg-dark-elevatedSurface h-16 lg:h-24 transition-colors duration-300 ease-linear"
    :class="{ shadow: !onTop, 'shadow-reverse': onTop }"
  >
    <div
      class="container relative mx-auto px-4 flex items-center justify-between lg:py-6 h-full"
    >
      <NuxtLink to="/">
        <img v-if="$config.logo" class="h-10 max-w-full" :src="$config.logo" />

        <template v-else-if="$config.logoLight && $config.logoDark">
          <img
            :src="$config.logoLight"
            class="h-10 max-w-full light-img"
            height="100"
          />
          <img
            :src="$config.logoDark"
            class="h-10 max-w-full dark-img"
            height="100"
          />
        </template>

        <AppLogo
          v-else-if="$options.components['AppLogo']"
          class="text-light-onSurfacePrimary dark:text-dark-onSurfacePrimary transition-colors duration-300 ease-linear"
        />
      </NuxtLink>

      <div v-if="$config.showSocialIconsOnHeader" class="dark:text-white flex">
        <a
          v-if="$config.linkedinUsername"
          :href="`https://www.linkedin.com/in/${$config.linkedinUsername}`"
          target="_blank"
          class="hidden sm:block ml-3"
          rel="noopener"
          aria-label="linkedin"
        >
          <IconLinkedin class="w-6 h-6" />
        </a>

        <a
          v-if="$config.twitterUsername"
          :href="`https://twitter.com/${$config.twitterUsername}`"
          target="_blank"
          class="hidden sm:block ml-8"
          rel="noopener"
          aria-label="twitter"
        >
          <IconTwitter class="w-6 h-6" />
        </a>

        <a
          v-if="$config.githubOwner"
          :href="`https://github.com/${$config.githubOwner}`"
          target="_blank"
          class="hidden sm:block ml-3"
          rel="noopener"
          aria-label="github"
        >
          <IconGithub class="w-8 h-8" />
          <a
          v-if="$config.githubOwner"
          :href="`https://github.com/${$config.githubOwner}`"
          target="_blank"
          class="hidden sm:block ml-3"
          rel="noopener"
          aria-label="github"
        >
          <IconGithub class="w-8 h-8" />
        </a>
      </div>
    </div>
  </header>
</template>

<script>
export default {
  data() {
    return {
      onTop: true,
    }
  },
  mounted() {
    window.addEventListener('scroll', this.handleScroll)
  },
  destroyed() {
    window.removeEventListener('scroll', this.handleScroll)
  },
  methods: {
    handleScroll() {
      this.onTop = window.pageYOffset < 60
    },
  },
}
</script>
```

### What is actually happening?

TheHeader.vue Component Not Working
![Grey Software](https://user-images.githubusercontent.com/53600644/107794898-d364c880-6d79-11eb-8d3e-319e478eb7d0.png)

**Comment by ArsalaBangash** [comment]
To clarify, TheHeader.vue is not displaying two Github links as per the code

**Comment by ArsalaBangash** [comment]
I would propose changing the issue name to "Overriding the default TheHeader.view component does not work"

**Comment by Tammo0987** [comment]
Is there any update on this? Or have we to wait for #726 to get this feature?

**Comment by akshgpt7** [comment]
For anyone looking how to override components of the theme:

So basically, you need to create a `components/` folder in the root of your project. Inside it, add the components you want to override with the same name as the original (ignore sub-directories). For example, `components/AppNav.vue` will override the `AppNav.vue` component of the theme.

In your `nuxt.config.js` file:

```js
export default theme({
  components: [
    './components', // default level is 0
    { path: 'node_modules/@nuxt/content-theme-docs/src/components/', level: 1 }
  ],
  docs: {
    primaryColor: "#33cb9a",
   .
   .
   .
  },
});
```

**Comment by ROBJkE** [comment]
> For anyone looking how to override components of the theme:
> 
> So basically, you need to create a `components/` folder in the root of your project. Inside it, add the components you want to override with the same name as the original (ignore sub-directories). For example, `components/AppNav.vue` will override the `AppNav.vue` component of the theme.
> 
> In your `nuxt.config.js` file:
> 
> ```js
> export default theme({
>   components: [
>     './components', // default level is 0
>     { path: '~~/node_modules/@nuxt/content-theme-docs/src/components/', level: 1 }
>   ],
>   docs: {
>     primaryColor: "#33cb9a",
>    .
>    .
>    .
>   },
> });
> ```

Thanks for this.  
Unfortunitly it doent work for me. I also followed [the docs](https://github.com/nuxt/components#overwriting-components).  
But it's not working.

My `nuxt.config.js` looks exactly like yours:
``` js
export default theme({
  ...
  components: [
    './components', // default level is 0
    { path: '~~/node_modules/@nuxt/content-theme-docs/src/components/', level: 1 }
  ],
  ...
});
```

My file is located in `my-project/components/AppFooter.vue` but when i run `npm run dev` nothing happens.  

Interessting: If i set `components: false` the whole components from docs-theme dissapear as it's expected.  
But thats the only thing that works.

**Comment by NozomuIkuta** [comment]
This issue is duplicate of #328.
While @ROBJkE provides a solution, I would like to continue further discussion there.

Let me close this issue. 🙏

**Comment by jculverwell** [comment]
Just wanted to confirm that @[ROBJkE](https://github.com/ROBJkE) instructions work.  I wanted to override the AppFooter

1. Create a folder called `components` at the root of the project
2. Copy `AppFooter.vue` from `\node_modules\@nuxt\content-theme-docs\src\components\app` to the `components` folder you just created
3. Modify `AppFooter.vue` as desired
4. I needed to rerun `npm run dev` for my changes to appear
5. As  @[ROBJkE](https://github.com/ROBJkE) noted, I did not need to modify `nuxt.config`
